src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  background-color: $elements-bg-color;
+  color: $main-text-color;
+
+  transition: $transition-time;
+
+      &:hover {
+        transform: scale(1.2);
+        z-index: 1;
+
+        .card__name {
+          color: $hover-text-color;
+        }
+      }
+
+  &__img {
+    display: block;
+    object-fit: contain;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $secondary-text-color;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-text-color;
+
+    &-wrapper {
+      display: flex;
+      justify-content: space-between;
+      align-items: end;
+      margin-bottom: 24px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary-text-color;
+    }
+
+    &-value {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: bold;
+      color: $main-text-color;
+    }
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
```

Comment:
It would be great to reset it for all elements

Reply:
Ok, I got it! It was not clear in the File changed tab.

Reply:
Oh no, that is the previous test remained in the logs, I have to cancel the commit